### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Einere**
## Description

This PR is modify some docs sections. 
- move some wrong paragraphs to appropriate section.
- add installation guide
- fix typo in example code
- add simple philosophy section

## Test Plan

no test plan

## Related Issues

#98 

## Tests

I added the following tests:

- no tests

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test:all` and make sure nothing fails. You can run `yarn test -u` to update snapshots if needed.
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by Einere** [comment]
@luke-hanwook 
I need your help about filling out installation guide in more environment.
how can i collaborate with you?

File: docs/src/overview/readme.stories.mdx [hunk 1]
```diff
@@ -35,6 +35,10 @@ import {Meta} from '@storybook/addon-docs/blocks';
 
 We love [react-hooks](https://code.fb.com/open-source/react-hooks) and willing to share our [react-native](https://facebook.github.io/react-native) ui components built in [function components](https://reactjs.org/docs/components-and-props.html#function-and-class-components) using hooks. Also the `dooboo-ui` is built on top of our favorite stacks like [emotion](https://emotion.sh/docs/@emotion/native), [typescript](https://www.typescriptlang.org/), [jest](https://jestjs.io), [react-testing-library](https://testing-library.com/docs/react-testing-library/intro), [expo](https://expo.io), [storybook](https://storybook.js.org), and so on.
 
+## Philosophy
+
+We are aiming to making `dooboo-ui` to be more user friendly and customizable, powerful library.
```
Comment (hyochan) [inline docs/src/overview/readme.stories.mdx:40]: I think you might want to change this sentence more fluent.
`dooboo-ui` aims to provide easy, lightweight, and flexible UI components?

Comment (Einere) [reply docs/src/overview/readme.stories.mdx:40]: oh, thank you for comment! i will fix it.

**Comment by luke-hanwook** [comment]
@Einere
I think we can work together at this PR 😄

**Comment by luke-hanwook** [comment]
@Einere 
why don't you write guide for [dooboo-cli](https://github.com/dooboolab/dooboo-cli) first.
I succeed installation all at my app.

**Comment by Einere** [comment]
@luke-hanwook 
oh, i didn't aware of that CLI. 😅
but, many users will not using `dooboo-cli`, because it force user to using there own boilerplate code. we need to provide freedom what to choose they want.
so, it would better describe popular method(ex expo or react-native-cli), and add tiny information how to use `dooboo-ui` with `dooboo-cli`. 😉

**Comment by Einere** [comment]
@luke-hanwook 
i added `dooboo-cli` info!

**Comment by luke-hanwook** [comment]
@Einere 
Nice work!
I will add document for web.

**Comment by luke-hanwook** [comment]
@Einere 
I think you should complete this PR.
I will open new PR for web.

**Comment by Einere** [comment]
@hyochan 
Can you review **Deploy Preview Hosting on PR / build_and_preview (pull_request)** GitHub Actions?
I think because there is no `FIREBASE_SERVICE_ACCOUNT_DOOBOOLAB_2F044` secret. [Plz see the yaml file](https://github.com/dooboolab/dooboo-ui/blob/5153a691ef7fb927ec42932eba139256d211f983/.github/workflows/deploy-docs-staging.yml#L37)
i have no permission to access secret, plz check it. 👏

**Comment by Einere** [comment]
@luke-hanwook 
yes, i will complete this PR. thank!

**Comment by yujong-lee** [comment]
@Einere It won't work for forked repository. Because like you mentioned, it can be accessed only here now.
So as I know, there is no other way than for the maintainer to create a new branch here and make a PR same as yours.

We'll look at it by applying your PR in local and run it. If you have better idea, please let us know.

**Pull request closed without merging**